v1/lib/core/Site.js

```diff
@@ -107,6 +107,21 @@ class Site extends React.Component {
               }}
             />
           )}
+          {this.props.config.algolia && (
+            <script
+              dangerouslySetInnerHTML={{
+                __html: `
+                document.addEventListener('keyup', function(e) {
+                  // keyCode for '/' (slash)
+                  if (e.keyCode === 191) {
+                    var search = document.getElementById('search_input_react');
```

Comment:
Use `const search` since it's not being reassigned.

Reply:
Nope these scripts do not get processed through Babel, so we have to be careful here. `const` has actually existed since ES5 so we're fine 😄